apps/fabric-website/index.html

```diff
@@ -47,6 +47,7 @@
     var scripts = [
       '//cdnjs.cloudflare.com/ajax/libs/react/16.3.1/umd/react.production.min.js',
       '//cdnjs.cloudflare.com/ajax/libs/react-dom/16.3.1/umd/react-dom.production.min.js',
+      // To serve minified files locally for testing purposes, change the next line to './dist/fabric-sitev5.min.js'
```

Comment:
This works for sure, but is pretty hacky. We could consider adding something like this to make it a little easier:

```js
var isLocal = window.location.hostname === 'localhost';
var appPath = isLocal ? host + 'dist/' : '';

var scripts = [
  '//cdnjs.cloudflare.com/ajax/libs/react/0.14.7/react.js',
  '//cdnjs.cloudflare.com/ajax/libs/react/0.14.7/react-dom.js',
  appPath + 'fabric-sitev5.min.js'
];
```